flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/typeutils/RowTypeInfo.scala

```diff
@@ -25,38 +25,25 @@ import org.apache.flink.api.scala.typeutils.CaseClassTypeInfo
 
 import scala.collection.mutable.ArrayBuffer
 import org.apache.flink.api.common.typeutils.TypeSerializer
-import org.apache.flink.api.table.{Row, TableException}
+import org.apache.flink.api.table.Row
 
 /**
  * TypeInformation for [[Row]].
  */
-class RowTypeInfo(fieldTypes: Seq[TypeInformation[_]], fieldNames: Seq[String])
+class RowTypeInfo(fieldTypes: Seq[TypeInformation[_]])
   extends CaseClassTypeInfo[Row](
     classOf[Row],
     Array(),
     fieldTypes,
-    fieldNames)
+    Nil)
```

Comment:
I would like to replace `Nil` with `for (i <- fieldTypes.indices) yield "f" + i`, so that we can keep fieldNames as `val`.